### ISSUE TITLE
Gravatar hovercard opening on hover from account gravatar in admin bar

### ISSUE THREAD

**Issue by kriskarkoski** [Automattic/wp-calypso#49831]
### Steps to reproduce the behavior

1. Go to WP Admin
2. Hover over Gravatar at the top-right
3. Note a Gravatar hovercard appears

#### What I expected to happen

The gravatar to reflect the hover state indicating it's a link to wordpress.com/me, but not open the hovercard.

#### What actually happened

A gravatar hovercard opens

#### Browser / OS version

Chrome 87.0.4280.141, OS X 

#### Does this happen on simple or atomic sites or both?

Both

#### Is there any console output or error text?

No

#### Level of impact (Does it block purchases? Does it affect more than just one site?)

Low

#### Screenshot / Video: If applicable, add screenshots to help explain your problem.

![Screen Shot 2021-02-08 at 10 46 09 AM](https://user-images.githubusercontent.com/1900444/107244757-61e4ed00-69fc-11eb-8306-a5ce78a7f443.png)

**Comment by tjcafferkey** [comment]
@kriskarkoski I can reproduce this on accounts that have the nav unification feature disabled. Am I ok to remove the "Nav Unification" label from this issue?

**Comment by kriskarkoski** [comment]
@tjcafferkey Yep, fine to remove. I wasn't 100% sure it was related but wasn't aware of any other recent changes to the nav so included it to be safe.

**Comment by Robertght** [comment]
I can still replicate this.

**Comment by kriskarkoski** [comment]
I can still reproduce this.
